addons/website_forum/models/forum.py

```diff
@@ -703,7 +710,11 @@ def validate(self):
                 raise AccessError(_('%d karma required to validate a post.', post.forum_id.karma_moderate))
             # if state == pending, no karma previously added for the new question
             if post.state == 'pending':
-                post.create_uid.sudo().add_karma(post.forum_id.karma_gen_question_new)
+                post.create_uid.sudo()._add_karma(
+                    post.forum_id.karma_gen_question_new,
+                    post,
+                    _('Asking a question'),
```

Comment:
Global question: shoudnd't we use the past tense everywhere ? aka "Asked a question" ? or use present perfect everywhere, just to :socks: the language we use :)